boards/shields/adafruit_2_8_tft_touch_v2/Kconfig.defconfig

```diff
@@ -43,7 +43,14 @@ config LVGL_VDB_SIZE
 	default 64
 
 config LVGL_BITS_PER_PIXEL
-	default 24
+	default 16
```

Comment:
I moved examples to RGB565, but it may be better to leave Adafruit board in RGB666 and, e.g. change Buydisplay to RGB565 so that 2 examples are available.

Reply:
sounds like a good approach more coverage.

Reply:
done